src/backend/external_loop_closure_receiver.cpp

```diff
@@ -171,8 +222,24 @@ void ExternalLoopClosureReceiver::update(const DynamicSceneGraph& graph,
       continue;
     }
 
-    // to_id, from_id, to_T_from
-    callback(to_node.id, from_node.id, gtsam::Pose3(edge.pose.matrix()));
+    const auto to_ns = getAgentTimestamp(graph.getNode(to_node.id)).count();
+    const auto from_ns = getAgentTimestamp(graph.getNode(from_node.id)).count();
+    uint64_t lc_lockout_ns = config.lc_lockout_time * 1e9;
+    bool should_add = should_add_lc(added_loop_closures_,
+                                    from_ns,
+                                    to_ns,
+                                    edge.pose,
+                                    lc_lockout_ns,
+                                    config.lc_min_pose_discrepancy);
+
+    if (should_add) {
+      LOG(WARNING) << "Added LC";
```

Comment:
Take a look at the verbosity configs [here](https://github.com/MIT-SPARK/Hydra/blob/develop/include/hydra/utils/logging.h), I'm trying to not have a bunch of warnings pop up when things are running nominally (probably also when we reject loop closures too but I guess that's more worthy of a warning). The `GvdIntegrator` is a reasonable example of what that looks like (there are a couple smaller scale ones as well if you grep for `VerbosityConfig`)